CMakeLists.txt

```diff
@@ -24,6 +24,9 @@ option(EMIL_BUILD_TESTS "Enable build of the tests" ${EMIL_DEFAULTOPT})
 option(EMIL_BUILD_EXAMPLES "Enable build of the examples" ${EMIL_DEFAULTOPT})
 cmake_dependent_option(EMIL_ENABLE_COVERAGE "Enable compiler flags for code coverage measurements" Off "EMIL_BUILD_TESTS" Off)
 
+option(EMIL_BUILD_PROTOBUF "Enable build of protobuf library" On)
```

Comment:
I would extend the text to point-out the implications of disabling Protobuf. E.g. "when disabling echo-rpc will also be disabled". Or something along those lines.